src/Analysis/SharedUtils/AccessibilityUtils/AccessibilityCSS/Accessibility.css

```diff
@@ -0,0 +1,27 @@
+/* App-specific styles */
+
+@import "AccessibilityTeams.css";
+@import "AccessibilityGWAS.css";
+
+/* Global styles */
+
+:root {
+  --focus-outline-color: #007bff; /* Bright Blue */
+}
+
+.screen-reader-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+.ant-modal-content button:focus {
+  outline: 1px dashed var(--focus-outline-color);
+  outline-offset: 1px;
+}
```

Comment:
Please add a final line break here and in the other CSS files. Github adds a red warning icon to files missing this. 

Here's the reason why: https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

Reply:
Thank you! Files updated @jarvisraymond-uchicago